spec/spec.yml

```diff
@@ -4840,6 +4841,27 @@ parameters:
     type: string
     format: uuid
     required: false
+  layer:
```

Comment:
Is this used anywhere? I don't see it added to an endpoint in this PR. If it is used, the name should be changed from `project` to `layer` I think.